src/libcore/fmt/mod.rs

```diff
@@ -379,6 +410,8 @@ pub trait Display {
 ///
 /// The `Octal` trait should format its output as a number in base-8.
 ///
+/// The alternate flag, `#o`, adds a `0o` in front of the output.
```

Comment:
Hm, isn't just `#` the alternate flag not the full `#o` entity? (i.e. one adds the flag to the format specifier `o`)